assets/js/components/Collection/Collection.jsx

```diff
@@ -23,8 +23,9 @@ const Collection = ({ collection }) => {
           <figure className="image is-square">
             <img
               src={
-                representativeImage != null
-                  ? representativeImage + "/square/500,500/0/default.jpg"
+                representativeWork
+                  ? representativeWork.representativeImage +
```

Comment:
Mind if we use the new JS string template ticks here for app consistency?     `${representativeWork.representativeImage}/square/500,500/0/default.jpg`